CMakeLists.txt

```diff
@@ -77,6 +81,19 @@ endif()
 #
 # Optimizations
 #
+
+# CMAKE_INTERPROCEDURAL_OPTIMIZATION can be defined to override the default behaviour.
+# We keep DEFINED CMAKE_INTERPROCEDURAL_OPTIMIZATION unset = FALSE for Debug builds to save
+# build time during development at the cost of a bigger memory foodprint.
+# Note: This is has caused issues on Fedoa https://bugzilla.rpmfusion.org/show_bug.cgi?id=5829
+if(NOT DEFINED CMAKE_INTERPROCEDURAL_OPTIMIZATION AND NOT CMAKE_BUILD_TYPE STREQUAL "Debug" AND NOT OPTIMIZE STREQUAL "off")
```

Comment:
```suggestion
if(NOT DEFINED CMAKE_INTERPROCEDURAL_OPTIMIZATION AND NOT CMAKE_BUILD_TYPE STREQUAL "Debug" AND NOT OPTIMIZE)
```

Reply:
If optimize is not set, CMake shoukd fail below. It defaults to "portable"

Reply:
I guess using STREQUAL is correct here

Reply:
@Holzhaus Your opinion?

Reply:
I agree with @daschuer. Command line arguments in CMake have a type (you can check with `cmake -L`):

    OPTIMIZE:STRING=portable

So we shouldn't mix different types and use STREQUAL throughout.